packages/controls/src/lumino/tabpanel.ts

```diff
@@ -111,13 +111,14 @@ export class TabPanel extends Widget {
    */
   constructor(options: TabPanel.IOptions = {}) {
     super();
-    this.addClass('p-TabPanel');
+    this.addClass('jupyter-widget-TabPanel');
+    this.addClass('jupyter-widget-TabPanel');
```

Comment:
```suggestion
```

Reply:
(or did you mean to put in one of the deprecated syntaxes?

Reply:
Thanks for picking up on this.